src/scripts/main.js

```diff
@@ -5,4 +5,19 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  row.innerHTML = `
+    <td>${person.name}</td>
+    <td>${(person.sex === 'f') ? 'Female' : 'Male'}</td>
+    <td>${person.born}</td>
+    <td>${person.died}</td>
+    <td>${person.died - person.born}</td>
+    <td>${Math.ceil(person.died / 100)}</td>
+  `;
```

Comment:
```suggestion
  row.innerHTML = `
    <tr>
      <td>${person.name}</td>
      <td>${(person.sex === 'f') ? 'Female' : 'Male'}</td>
      <td>${person.born}</td>
      <td>${person.died}</td>
      <td>${person.died - person.born}</td>
      <td>${Math.ceil(person.died / 100)}</td>
    </tr>
  `;
```
You could just add a tag to your `innerHTML` and there would be no need in 'row' variable

Reply:
okay, thanks!